go.mod

```diff
@@ -15,6 +14,7 @@ require (
 	github.com/google/go-containerregistry v0.0.0-20190110221225-f514e780f7cd
 	github.com/gorilla/context v1.1.1 // indirect
 	github.com/gorilla/mux v1.6.2 // indirect
+	github.com/morikuni/aec v0.0.0-20170113033406-39771216ff4c // indirect
```

Comment:
Is this the new library that is used by docker?

Reply:
yup, it's mentioned in https://github.com/moby/moby/pull/38398